helpers/config.go

```diff
@@ -0,0 +1,25 @@
+package helpers
+
+import (
+	"os"
+	"path/filepath"
+)
+
+// GetConfigDirectory will return where config and state files should be
+// stored, either respecting `FLY_CONFIG_DIR` or defaulting to the user's home
+// directory at `~/.fly`.
+func GetConfigDirectory() (string, error) {
+	var dir string
+
+	if value, isSet := os.LookupEnv("FLY_CONFIG_DIR"); isSet {
```

Comment:
might be good to support the xdg spec env vars here too, while we're doing this. that could be done separately.

Reply:
https://clig.dev/#configuration